packages/eslint-config-custom/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "dependencies": {
     "eslint-config-prettier": "^8.3.0",
-    "eslint-config-turbo": "^1",
+    "eslint-config-turbo": "^2",
```

Comment:
Not sure about this one. Should probably not be part of the PR.

Reply:
Seems to be important. Idk why. Maybe because the turbo version got updated or so.